src/components/cartPage/howDiscoverPicker.jsx

```
@@ -58,79 +57,72 @@ function HowDiscoverPicker({
+            {pickedItem?.name === 'Other' && (
+              <>
+                <Field
+                  name="howDiscover"
+                  id="howDiscover"
+                  type="text"
+                  value={value}
+                  onChange={handleChange}
+                  className="how-discover-option"
+                  placeholder={lang('EnterYourOwnOption', language)}
+                />
+              </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)